docs/example-code.md

```diff
@@ -0,0 +1,164 @@
+### Example Code ###
```

Comment:
There are no links pointing to this file. We will revise content and then add the link in navbar once we are satisfied with the examples.

Reply:
This file needs to show examples of out of band loading of ads as well with vanilla js. Aka separate file at root that lists data-tagged elements and injects ads.